### DISCUSSION TITLE
How to color/highlight current cell?

### DISCUSSION THREAD

**Discussion by reagle** [saulpw/visidata#1466] [Q&A]
I'd like my current cell -- at the interselction of the highlighted column and row -- to be more salient. How to do so?

**Comment by frosencrantz** [comment]
Similar question: https://github.com/saulpw/visidata/discussions/1093
Also, this recent thread about coloring other cells, might be of interest: https://github.com/saulpw/visidata/issues/1474

**Reply by reagle** [reply]
Thank you, should I close this one then?

**Reply by frosencrantz** [reply]
Are either of these satisfactory solutions for you?  Maybe some of these suggestions will help you.

I don't yet have a solution that I'm happy with per https://github.com/saulpw/visidata/discussions/1093 .
I would like to have a very clear highlighting of the current cell.  Maybe a solution that works for you will also work for me, if the other suggestions don't help.

**Reply by saulpw** [reply]
Hi @reagle and @frosencrantz, try this:

```
TableSheet.colorizers += [
    CellColorizer(3, 'bold red on black', lambda s,c,r,v: c is s.cursorCol and r is s.cursorRow)
]
```
